governance.md

```diff
@@ -68,26 +66,29 @@ The foundations of Project governance are:
 -   Active Contribution
 -   Institutional Neutrality
 
-Traditionally, Project leadership was provided by a BDFL (Fernando Perez) and
-subset of Contributors, called Core Developers, whose active and consistent
-contributions have been recognized by their receiving “commit rights” to the
-Project GitHub repositories. In general all Project decisions are made through
-consensus among the Core Developers with input from the Community. The BDFL
-can, but rarely chooses to, override the Core Developers and make a final
-decision on a matter.
-
-While this approach has served us well, as the Project grows and faces more
-legal and financial decisions and interacts with other institutions, we see a
-need for a more formal governance model. Moving forward The Project leadership
-will consist of a BDFL and Steering Council. We view this governance model as
-the formalization of what we are already doing, rather than a change in
-direction.
+Project leadership was provided by the original author, John
+D. Hunter.  Shortly before his passing in 2012, leadership was
+transferred to Michael Droettboom, who later invited Thomas Caswell as
+a co-lead.  Additional leadership has also been provided by a subset
+of Contributors, called Core Developers, whose active and consistent
+contributions have been recognized by their receiving “commit rights”
```

Comment:
In this text, "Core Developers" refers to everyone in the "Developers" github team.  I think I should just change the text here.  The meaning of the "Core" team on github will probably need to be refined.  (Right now, it's sort of "Numfocus subcommittee" + regular meeting attendees, but that's not a definition I think we necessarily want to keep).

I think to move forward, I'll just correct the text here to match the github team name, and we can refine the definition of Core later if we want/need to.